package.json

```
@@ -62,6 +62,9 @@
   },
   "license": "UNLICENSED",
   "devDependencies": {
+    "@biomejs/biome": "^2.0.0",
+    "@biomejs/cli-darwin-arm64": "^2.0.0",
+    "@biomejs/cli-linux-x64-musl": "^2.0.0",
```

Biome recommends pinning the dependencies
```suggestion
    "@biomejs/biome": "2.0.0",
    "@biomejs/cli-darwin-arm64": "2.0.0",
    "@biomejs/cli-linux-x64-musl": "2.0.0",
```